blog/_posts/2016-07-15-sip-summary-minutes.md

```diff
@@ -2,7 +2,7 @@
 layout: blog
 post-type: blog
 by: Jorge Vicente Cantero
-title: A Reformed Scala Improvement Process
+title: SIP Committee Numbers 2 SIPs and Updates 3 Others
```

Comment:
After looking at this again, I'm beginning to think it would read better if it was: "SIP Meeting Results: 2 SIPs Numbered and 3 SIPs Updated" It's catchier. Otherwise I fear people might miss that this blog article is about the outcome of the meeting.